src/client/components/views/editView/utils.js

```diff
@@ -44,3 +44,14 @@ export const debounceUpdateValue = debounce((value, onUpdateValue) => {
     onUpdateValue(value);
   }
 }, 800);
+
+/**
+ *
+ * @param {*string} string
+ * @returns {spaced string e.g ACROText -> ACRO Text}
+ */
+export function insertSpaces(string) {
+  string = string.replace(/([a-z])([A-Z])/g, '$1 $2');
+  string = string.replace(/([A-Z])([A-Z][a-z])/g, '$1 $2');
+  return string;
```

Comment:
There's nothing wrong here but I would say it's a bit atypical to have a function modify the value of a parameter.

If I understood the goal correctly (camelCase to space case), I think something like the following would work as well (+ it also works if a string contains numbers):
```
return string.replace(/([a-z0-9])([A-Z])/g, '$1 $2');
``` 

I won't block the PR because of this. Just wanted you to know